.prettierignore

```diff
@@ -0,0 +1,29 @@
+# Default system files
+patches
+pnpm-lock.yaml
+
+# #181: Ignore all files currently linted via eslint/stylelint
+*.ts
+*.tsx
+*.cjs
+*.js
+*.jsx
+*.html
+*.css
+*.mdx
+*.vue
+
+# Build and dist directeories
```

Comment:
Doesn't prettier per default ignore everything in .gitignore?

Reply:
It seems to ignore the `.gitignore` files in subfolders, so I had to add them unfortunately :(